### PR TITLE
Update metric tables function name closes #172

### LINKED ISSUE #172: Update metric tables function name
**Describe the bug**

**Steampipe version (`steampipe -v`)**
Example: v0.3.0

**Plugin version (`steampipe plugin list`)**
Example: v0.5.0

**To reproduce**
Steps to reproduce the behavior (please include relevant code and/or commands).

**Expected behavior**
The table function name should contain the cloud provider name.

**Additional context**

### PR DESCRIPTION
**Pull request by ParthaI**
# Integration test logs
<details>
  <summary>Logs</summary>
  
```
Add passing integration test logs here
```
</details>

# Example query results
<details>
  <summary>Results</summary>
  
```
Add example SQL query results here (please include the input queries as well)
```
</details>

### CODE REVIEW

File: azure/plugin.go [hunk 1]
```diff
@@ -34,19 +34,19 @@ func Plugin(ctx context.Context) *plugin.Plugin {
...
-			"azure_compute_disk_metric_read_ops_daily":                    tableComputeDisksReadOpsMetricDaily(ctx),
-			"azure_compute_disk_metric_write_ops":                         tableComputeDisksWriteOpsMetric(ctx),
-			"azure_compute_disk_metric_write_ops_daily":                   tableComputeDisksWriteOpsMetricDaily(ctx),
-			"azure_compute_disk_metric_write_ops_hourly":                  tableComputeDisksWriteOpsMetricHourly(ctx),
+			"azure_compute_disk_metric_read_ops":                          tableAzureComputeDisksReadOpsMetric(ctx),
+			"azure_compute_disk_metric_read_ops_hourly":                   tableAzureComputeDisksReadOpsMetricHourly(ctx),
+			"azure_compute_disk_metric_read_ops_daily":                    tableAzureComputeDisksReadOpsMetricDaily(ctx),
+			"azure_compute_disk_metric_write_ops":                         tableAzureComputeDisksWriteOpsMetric(ctx),
+			"azure_compute_disk_metric_write_ops_daily":                   tableAzureComputeDisksWriteOpsMetricDaily(ctx),
+			"azure_compute_disk_metric_write_ops_hourly":                  tableAzureComputeDisksWriteOpsMetricHourly(ctx),
```
Comment (Subhajit97) [inline azure/plugin.go:42]:
```suggestion
			"azure_compute_disk_metric_read_ops":                          tableAzureComputeDiskReadOpsMetric(ctx),
			"azure_compute_disk_metric_read_ops_hourly":                   tableAzureComputeDiskReadOpsMetricHourly(ctx),
			"azure_compute_disk_metric_read_ops_daily":                    tableAzureComputeDiskReadOpsMetricDaily(ctx),
			"azure_compute_disk_metric_write_ops":                         tableAzureComputeDiskWriteOpsMetric(ctx),
			"azure_compute_disk_metric_write_ops_daily":                   tableAzureComputeDiskWriteOpsMetricDaily(ctx),
			"azure_compute_disk_metric_write_ops_hourly":                  tableAzureComputeDiskWriteOpsMetricHourly(ctx),
```

Comment (ParthaI) [reply azure/plugin.go]: Updated with format `tableAzure<service><resource>Metric<metric_type><hourly/daily>`

File: azure/plugin.go [hunk 2]
```diff
@@ -34,19 +34,19 @@ func Plugin(ctx context.Context) *plugin.Plugin {
...
 			"azure_compute_image":                                         tableAzureComputeImage(ctx),
 			"azure_compute_resource_sku":                                  tableAzureResourceSku(ctx),
 			"azure_compute_snapshot":                                      tableAzureComputeSnapshot(ctx),
 			"azure_compute_virtual_machine":                               tableAzureComputeVirtualMachine(ctx),
-			"azure_compute_virtual_machine_metric_cpu_utilization":        tableComputeInstanceCpuUtilizationMetric(ctx),
-			"azure_compute_virtual_machine_metric_cpu_utilization_daily":  tableComputeInstanceCpuUtilizationMetricDaily(ctx),
-			"azure_compute_virtual_machine_metric_cpu_utilization_hourly": tableComputeInstanceCpuUtilizationMetricHourly(ctx),
+			"azure_compute_virtual_machine_metric_cpu_utilization":        tableAzureComputeInstanceMetricCpuUtilization(ctx),
+			"azure_compute_virtual_machine_metric_cpu_utilization_daily":  tableAzureComputeInstanceMetricCpuUtilizationDaily(ctx),
+			"azure_compute_virtual_machine_metric_cpu_utilization_hourly": tableAzureComputeInstanceMetricCpuUtilizationHourly(ctx),
```
Comment (Subhajit97) [inline azure/plugin.go:49]:
```suggestion
			"azure_compute_virtual_machine_metric_cpu_utilization":        tableAzureComputeVirtualMachineMetricCpuUtilization(ctx),
			"azure_compute_virtual_machine_metric_cpu_utilization_daily":  tableAzureComputeVirtualMachineMetricCpuUtilizationDaily(ctx),
			"azure_compute_virtual_machine_metric_cpu_utilization_hourly": tableAzureComputeVirtualMachineMetricCpuUtilizationHourly(ctx),
```

File: azure/plugin.go [hunk 3]
```diff
@@ -34,19 +34,19 @@ func Plugin(ctx context.Context) *plugin.Plugin {
 			"azure_compute_availability_set":                              tableAzureComputeAvailabilitySet(ctx),
 			"azure_compute_disk":                                          tableAzureComputeDisk(ctx),
 			"azure_compute_disk_encryption_set":                           tableAzureComputeDiskEncryptionSet(ctx),
-			"azure_compute_disk_metric_read_ops":                          tableComputeDisksReadOpsMetric(ctx),
-			"azure_compute_disk_metric_read_ops_hourly":                   tableComputeDisksReadOpsMetricHourly(ctx),
-			"azure_compute_disk_metric_read_ops_daily":                    tableComputeDisksReadOpsMetricDaily(ctx),
-			"azure_compute_disk_metric_write_ops":                         tableComputeDisksWriteOpsMetric(ctx),
-			"azure_compute_disk_metric_write_ops_daily":                   tableComputeDisksWriteOpsMetricDaily(ctx),
-			"azure_compute_disk_metric_write_ops_hourly":                  tableComputeDisksWriteOpsMetricHourly(ctx),
+			"azure_compute_disk_metric_read_ops":                          tableAzureComputeDisksMetricReadOps(ctx),
```
Comment (Subhajit97) [inline azure/plugin.go:37]:
```suggestion
			"azure_compute_disk_metric_read_ops":                          tableAzureComputeDiskMetricReadOps(ctx),
```

**Review by Subhajit97 — APPROVED** [review]

**Review by cbruno10 — APPROVED** [review]

**Pull request merged**